DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -183,6 +183,7 @@ private static void RegisterServices(IServiceCollection services)
             services.AddScoped<ISessionService, SessionService>();
             services.AddScoped<ISupervisorService, SupervisorService>();
             services.AddScoped<IUserService, UserService>();
+            services.AddScoped<IGroupsService, GroupsService>();

```

Comment:
I was tempted to ask you to move this alphabetically since I tidied them all up, but I know there is no way it'll stay like that 😢